cinema_service/settings.py

```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-ru6ndfka@95_(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy"
-)
+SECRET_KEY = ("django-insecure-ru6ndfka@95_"
+              "(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy")
```

Comment:
The `SECRET_KEY` is currently hardcoded and exposed in the settings file. This is a security risk, especially in production environments. Consider using environment variables or a separate configuration file to securely manage the `SECRET_KEY`.